stencil/src/components/pulumi-api-symbol/pulumi-api-symbol.scss

```diff
@@ -0,0 +1,26 @@
+.symbol {
+  height: 24px;
+  width: 24px;
+  font-size: 16px;
+  border-width: 1px;
+  border-style: solid;
+  box-sizing: border-box;
+  border-radius: 50%;
+  margin-right: 6px;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  font-weight: 700;
+}
+
+.symbol-small {
+  height: 14px;
```

Comment:
so tiny, totally not blocking at all, but my personal preference for stuff like this is to throw it in a variable, so that future us can look at the file in a glance and see `$symbol-small-size` and `$symbol-small-large` and understand the options.

Reply:
Good call out 👍